docs/usage/configuration-options.md

```diff
@@ -79,9 +79,12 @@ With the above config:
 
 ## additionalBranchPrefix
 
-This value defaults to an empty string, and is typically not necessary.
-Some managers previously populated this field, but they no longer do so by default.
-You normally don't need to configure this, but one example where it can be useful is combining with `parentDir` in monorepos to split PRs based on where the package definition is located, e.g.
+By default, the value for this config option is an empty string.
+Normally you don't need to set this config option.
+
+Here's an example where `additionalBranchPrefix` can help you.
+Say you're using a monorepo and want to split pull requests based on the location of the package definition.
+This can be done with this configuration:
```

Comment:
I don't understand our current example. It's missing some key information for me:

- _Why_ a user may want to split pull requests based on the location of the package definition? What's "wrong" with the default behavior?
- What does the new PR title or branch name look like?

Reply:
In a monorepo, many teams can share one repo. Thus they divide work by subfolders. So if you want Renovate PRs not to "span" multiple teams, you could want to separate them by subfolder.